Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -351,11 +353,17 @@ private void Update()
             elapsed = 0f;
         }
 
+        if(currentOverlay != "None" && currentLayer != WorldController.Instance.cameraController.CurrentLayer)
```

Comment:
Stylecop `SA1000 : CSharp.Spacing : The spacing around the keyword 'if' is invalid. (Assembly-CSharp)`